jarvis.go

```diff
@@ -2,23 +2,17 @@ package main
 
 import (
 	"github.com/gin-gonic/gin"
+	"github.com/magrathealabs/jarvis/domain/repositories"
 	"github.com/magrathealabs/jarvis/infrastructure/handlers"
 	"github.com/magrathealabs/jarvis/libs/env"
-	"github.com/magrathealabs/jarvis/libs/handler"
 )
 
-func usedHandlers() []handler.Handler {
-	return []handler.Handler{
-		handlers.NewRootHandler(),
-	}
-}
+var metricRepository repositories.MetricRepository
```

Comment:
Why is this outside any method scope?

Reply:
It's a main app. This is shared to all handlers and domain with a recursive call. But, here i can configure as "constant".